google_drive/google_calendar.py

```diff
@@ -1,33 +1,35 @@
+"""
+Google Calendar XBlock implementation
+"""
 # -*- coding: utf-8 -*-
 #
 
 # Imports ###########################################################
-
-import pkg_resources
-#import logging
-import textwrap
-#import json
-#import webob
-#from lxml import etree
-#from xml.etree import ElementTree as ET
+import logging
 
 from xblock.core import XBlock
-from xblock.fields import Scope, String
+from xblock.fields import Scope, String, Integer
 from xblock.fragment import Fragment
 
-# from StringIO import StringIO
-
-from .utils import render_template, AttrDict, load_resource
+from xblockutils.publish_event import PublishEventMixin
+from xblockutils.resources import ResourceLoader
 
+LOG = logging.getLogger(__name__)
+RESOURCE_LOADER = ResourceLoader(__name__)
 
-# Globals ###########################################################
 
-#log = logging.getLogger(__name__)
+# Constants ###########################################################
+DEFAULT_CALENDAR_ID = "edx.org_lom804qe3ttspplj1bgeu1l3ak@group.calendar.google.com"
+CALENDAR_TEMPLATE = "/templates/html/google_calendar.html"
+CALENDAR_EDIT_TEMPLATE = "/templates/html/google_calendar_edit.html"
+CALENDAR_IFRAME = (
+    '<iframe src="https://www.google.com/calendar/embed'
+    '?mode={}&amp;src={}&amp;showCalendars=0" title="{}"></iframe>'
```

Comment:
This template is hard to understand with positional fields. Naming them would be better.